frontend/form-builder/src/components/SettingsSidebar/SettingsSidebar.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import classes from "./SettingsSidebar.module.css";
+import { Heading, Textfield } from "@digdir/design-system-react";
+
+export const SettingsSidebar = (): React.JSX.Element => {
+  return (
+    <div className={classes.sidebar}>
+      <Heading level={2} size="medium" spacing>Settings</Heading>
+      <Textfield label="Component name" size="small" className={classes.spacing}/>
+      <Textfield label="Minimum length" size="small" className={classes.spacing}/>
+      <Textfield label="Maximum length" size="small" error="Maximum length not set" className={classes.spacing}/>
```

Comment:
Foreløpig vil vel denne alltid vise error "Maximum length not set"? 😊

Reply:
Ja, det var mest for å vise error-beskjeden med tanke på layout. Men det hadde kanskje vært nok å vist det på skjermbildet 😊